### ISSUE TITLE
Linking to Registry About page

### ISSUE THREAD

**Issue by bill-anderson** [devinit/D-Portal#411]
I had a request at the MA (from World Vision International) which makes sense to me ...

Can we somehow link to (or display) the [about page](https://www.iatiregistry.org/publisher/about/aai) from the registry?

Not particularly bothered how or where, so long as it is clear and simple.

**Comment by notshi** [comment]
Sure, but thinking about it - it would be great if the Registry allowed you to link to its various pages using the IATI identifier rather than the CKAN slug.

This could easily be done as a redirect on top of the current server; ie. these two links would be the same page.

1. ```https://www.iatiregistry.org/publisher/about/aai``` 
2. ```https://www.iatiregistry.org/publisher/about/NL-KVK-27264198```

Currently, we have to maintain a mapping between these two identifiers and that makes it harder for anyone to link direct to the registry from IATI data.

Furthermore, it's a complicated process to download all these ids from the Registry - you need to visit the individual pages of each publisher (thousands of hits on the website) just to build this id map before you can link to the site.

In short, yes, this can be done and it will be done but we believe the suggestion above would benefit everyone.

**Comment by notshi** [comment]
Hi @bill-anderson - this is now live on [d-portal](http://d-portal.org/ctrack.html?publisher=NL-KVK-27264198#view=main).

![pub_link](https://user-images.githubusercontent.com/1515961/37541597-117c645c-2953-11e8-91ad-7ee34c4e51eb.png)

This is also in the footer section under the contact link.
![sub](https://user-images.githubusercontent.com/1515961/37541983-718bccce-2954-11e8-892e-cc31074bd2bb.png)
